docs/manuals/core/entities/scheduling-and-job-execution.md

```diff
@@ -155,6 +156,57 @@ This example will produce the following output:
 (3) Number of job: 0.
 ```
 
+## Cancel Job
+
+Jobs are created when a task is submitted.
+
+- You can cancel the Job with `taipy.cancel_job(job)^`. When cancelling a Job, you will effectively set the `Status^` of subsequent jobs of the cancelled Job to `ABANDONED`.
+
+Cancelling a `RUNNING` Job will set the Job to `CANCELLED`. However, it will not terminate the subprocess that has already been dispatched to execute the Job.
```

Comment:
```suggestion
!!! Note
    Cancelling a `RUNNING` Job will set the Job to `CANCELLED`. However, it will not terminate 
    the subprocess that has already been dispatched to execute the Job.
```

Reply:
@jrobinAV Do you think a warning would be better?